tests/test_feedbacks.py

```diff
@@ -33,19 +33,31 @@
 @pytest.fixture
 def restore_resolution_registers(motion_controller):
     mc, alias = motion_controller
-    pp = mc.communication.get_register(PAIR_POLES_REGISTER, servo=alias)
-    inc_res_1 = mc.communication.get_register(INCREMENTAL_RESOLUTION_1_REGISTER, servo=alias)
-    inc_res_2 = mc.communication.get_register(INCREMENTAL_RESOLUTION_2_REGISTER, servo=alias)
-    abs_1_1_sing_turn = mc.communication.get_register(ABS1_1_SINGLE_TURN_REGISTER, servo=alias)
-    abs_1_2_sing_turn = mc.communication.get_register(ABS1_2_SINGLE_TURN_REGISTER, servo=alias)
-    abs_2_1_sing_turn = mc.communication.get_register(ABS2_1_SINGLE_TURN_REGISTER, servo=alias)
+    registers = [
+        PAIR_POLES_REGISTER,
+        INCREMENTAL_RESOLUTION_1_REGISTER,
+        INCREMENTAL_RESOLUTION_2_REGISTER,
+        ABS1_1_SINGLE_TURN_REGISTER,
+        ABS1_2_SINGLE_TURN_REGISTER,
+        ABS2_1_SINGLE_TURN_REGISTER,
+    ]
+    registers_values = [
+        mc.communication.get_register(register, servo=alias)
+        if mc.info.register_exists(register, servo=alias)
+        else None
+        for register in registers
+    ]
     yield
-    mc.communication.set_register(PAIR_POLES_REGISTER, pp, servo=alias)
-    mc.communication.set_register(INCREMENTAL_RESOLUTION_1_REGISTER, inc_res_1, servo=alias)
-    mc.communication.set_register(INCREMENTAL_RESOLUTION_2_REGISTER, inc_res_2, servo=alias)
-    mc.communication.set_register(ABS1_1_SINGLE_TURN_REGISTER, abs_1_1_sing_turn, servo=alias)
-    mc.communication.set_register(ABS1_2_SINGLE_TURN_REGISTER, abs_1_2_sing_turn, servo=alias)
-    mc.communication.set_register(ABS2_1_SINGLE_TURN_REGISTER, abs_2_1_sing_turn, servo=alias)
+    for register, register_value in zip(registers, registers_values):
+        if register_value is not None:
+            mc.communication.set_register(register, register_value, servo=alias)
+
+
+def skip_if_QEI2_is_not_available(mc, alias, sensor=SensorType.QEI2):
```

Comment:
The function name should be lowercase. Can you make it a fixture, to be coherent with other skip fixtures?